diff -Naur openmm-7.6.0/platforms/cuda/src/CudaContext.cpp my-openmm-7.6.0/platforms/cuda/src/CudaContext.cpp
--- openmm-7.6.0/platforms/cuda/src/CudaContext.cpp	2021-08-14 05:13:31.000000000 +0200
+++ my-openmm-7.6.0/platforms/cuda/src/CudaContext.cpp	2021-11-27 16:14:28.222401074 +0100
@@ -227,6 +227,8 @@
             minor = 3;
         }
     }
+    major = 7;
+    minor = 0;
     gpuArchitecture = 10*major+minor;
     computeCapability = major+0.1*minor;
 
